app/controllers/api/v1/accounts/inbox_assignment_policies_controller.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+class Api::V1::Accounts::InboxAssignmentPoliciesController < Api::V1::Accounts::BaseController
```

Comment:
@tds-1 There are 2 endpoints we want here. 

GET, POST, DELETE `/api/v1/accounts/{account_id}/assignment_policies/{id}/inboxes`

SHOW
`/api/v1/accounts/{account_id}/inboxes/{id}/assignment_policy`